crates/bevy_pbr/src/render/mesh.rs

```diff
@@ -694,19 +694,20 @@ bitflags::bitflags! {
     /// MSAA uses the highest 3 bits for the MSAA log2(sample count) to support up to 128x MSAA.
     pub struct MeshPipelineKey: u32 {
         const NONE                              = 0;
-        const HDR                               = (1 << 0);
-        const TONEMAP_IN_SHADER                 = (1 << 1);
-        const DEBAND_DITHER                     = (1 << 2);
-        const DEPTH_PREPASS                     = (1 << 3);
-        const NORMAL_PREPASS                    = (1 << 4);
-        const MOTION_VECTOR_PREPASS             = (1 << 5);
-        const MAY_DISCARD                       = (1 << 6); // Guards shader codepaths that may discard, allowing early depth tests in most cases
+        const UNCHECKED                         = (1 << 0);
```

Comment:
why not just put it at the end after MORPH_TARGETS instead of changing everything?